pyproject.toml

```diff
@@ -9,6 +9,8 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.25.1"
+pytest = "^6.2.4"
+selenium = "^3.141.0"
```

Comment:
Note that we could add these as dev-dependencies like you have done pylint

Reply:
Good call. Updated in https://github.com/BenjaminEHowe/todo-apprenticeship/commit/965e1b492ab00db3a02febc422e04e96fb681136.